app/lib/methods/helpers/debounce.ts

```diff
@@ -21,3 +23,7 @@ export function debounce(func: Function, wait?: number, immediate?: boolean) {
 	_debounce.stop = () => clearTimeout(timeout!);
 	return _debounce;
 }
+
+export function useDebounce(func: (...args: any) => any, wait?: number): (...args: any[]) => void {
```

Comment:
IMHO, I think we can export `useDebounce` from libs/hooks, WDYT?

Reply:
The question here is more to maintain debounce things on the same local.
If we change the useDebounce to hooks, we will have two files doing the same thing.